src/routes/api/profile.routes.js

```diff
@@ -0,0 +1,14 @@
+import express from 'express';
+import controllers from '../../controllers';
+import { profileReqCheckpoint, validatProfilUpdate } from '../../middlewares/profileDataChecker';
+import { authorizeAccess, signupValidation } from '../../middlewares/authValidation';
+
+const profileRouter = express.Router();
+const { displayUserProfile, updateUserProfile, changeUserPassword } = controllers.ProfileController;
+
+profileRouter.get('/:requestedProfile', authorizeAccess, profileReqCheckpoint, displayUserProfile);
+profileRouter.get('/', authorizeAccess, profileReqCheckpoint, displayUserProfile);
+profileRouter.patch('/update-profile', authorizeAccess, validatProfilUpdate, profileReqCheckpoint, signupValidation, updateUserProfile);
+profileRouter.patch('/change-password', authorizeAccess, validatProfilUpdate, profileReqCheckpoint, changeUserPassword);
```

Comment:
here you can change and use /password instead of /change-password

Reply:
@mifeille I have changed this